moped-etl/ecapris-statuses/process/queries.py

```diff
@@ -0,0 +1,54 @@
+"""
+Queries for FSD Data Warehouse
+
+Oracle does not store trailing zeroes on numbers, but we need them to match eCapris subproject IDs in Moped DB which have them.
+Ex. eCapris ID 10553.030 is stored as 10553.030 in Moped DB, but stored as 10553.03 in the Oracle DB.
+So, we format the SP_NUMBER to have 3 decimal places using its defined Oracle type Number(18,3).
+'FM999999999999999.000' formats the number to have 3 decimal places with trailing zeroes and 18 digits total.
+"""
+
+ORACLE_QUERIES = {
+    "subproject_statuses": """
+    SELECT
+        TO_CHAR(SP_NUMBER, 'FM999999999999999.000') AS SP_NUMBER,
```

Comment:
I just updated this to fix a bug that I caught when reviewing statuses with Nathan this afternoon where trailing zeroes like in `10553.030` or `11280.030` in subproject numbers were being lost in the Oracle DB query since they are stored as numbers in their database, but we store them as strings in the `moped_project` table.

Reply:
nice catch! that's kinda wild that they're incorrectly stored as numbers in that DB

Reply:
yeah, so glad we ran into this by chance and very strange indeed but fun to learn a little more about Oracle DBs 📚